### ISSUE TITLE
Update ember-cli-babel dependency to > 7.26.6

### ISSUE THREAD

**Issue by sly7-7** [ef4/ember-elsewhere#54]
Hello, 

I encounter the following deprecation warning: 
```
### Details ###

Prior to v7.26.6, ember-cli-babel sometimes transpiled imports into the equivalent Ember Global API, potentially triggering this deprecation message indirectly, even when you did not observe these deprecated usages in your code.

The following outdated versions are found in your project:

* ember-cli-babel@6.18.0, currently used by:
  * ember-elsewhere@1.2.1
    * Depends on ember-cli-babel@^6.16.0
```

https://github.com/ef4/ember-elsewhere/blob/a8d034d38c3090de3f4105aa29ec14d4ef700bb0/package.json#L29

Would it be possible to update it, please ?

**Comment by cah-briangantzler** [comment]
Working on a PR now, but having trouble submitting it https://github.com/ef4/ember-elsewhere/issues/55

**Comment by sly7-7** [comment]
@cah-briangantzler seems like #56 is superseded by #57 to tackle this issue (and the whole upgrade process).

**Comment by cah-briangantzler** [comment]
Great, looking forward to the update. Trying to get rid of the noise in my builds for the Ember global deprecation

**Comment by ef4** [comment]
Fixed and released in 2.0

**Comment by cah-briangantzler** [comment]
Thanks for releasing 2.0
